### PR TITLE
Notify new issues

### PR DESCRIPTION
**Pull request by bangtoven**
### _How did you test your changes?_

image

### CODE REVIEW

File: .github/workflows/notification.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  pull_request:
+    types: [opened, reopened]
+  issues:
+    types: [opened, reopened]
+
+env:
+  CHANNEL_WALLET_FEEDBACK: ${{ vars.CHANNEL_WALLET_FEEDBACK }}
+  CHANNEL_WALLET_SQUAD_BUILD: ${{ vars.CHANNEL_WALLET_SQUAD_BUILD }}
+  ON_CALL_WALLET: ${{ vars.ON_CALL_WALLET }}
+  ON_CALL_BUILD_SQUAD: ${{ vars.ON_CALL_BUILD_SQUAD }}
```
Comment (bangtoven) [inline .github/workflows/notification.yml:13]: image

Comment (bangtoven) [reply .github/workflows/notification.yml:13]: currently they point to a test channel / user group

File: .github/workflows/notification.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    #     if [ "$permission" != "admin" ] && [ "$permission" != "write" ]; then
+    #       echo "::notice:: User $USER has $permission permission."
+    #       exit 0
+    #     fi
+    #     echo "skip=true" >> $GITHUB_ENV
+    #     echo "::error:: User $USER has $permission permission. No slack alert required."
+    #   env:
+    #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    #     REPO: ${{ github.repository }}
+    #     USER: ${{ github.actor }}
```
Comment (cb-jake) [inline .github/workflows/notification.yml:38]: do we need this commented code still?

Comment (bangtoven) [reply .github/workflows/notification.yml:38]: @cb-jake yes. i should have left some comments there...
this is to skip alerting when one of us opens a PR. will be reenabled after testing.

Comment (bangtoven) [reply .github/workflows/notification.yml:38]: https://coinbase.slack.com/archives/C04M9D8D41W/p1675977215650019

**Review by cb-jake — APPROVED** [review]

**Pull request merged**